tokens/src/components/Form/other.json

```diff
@@ -2,11 +2,11 @@
   "other": {
     "form": {
       "control": {
-        "cursor": { "value": "auto", "source": "$custom-control-cursor" },
-        "range-track-cursor": { "value": "pointer", "source": "$custom-range-track-cursor" }
+        "cursor": { "value": "auto", "type": "strokeStyle", "source": "$custom-control-cursor" },
```

Comment:
I believe `strokeStyle` type is intended only for lines, for cursor we could probably use `cursorStyle` (it's not in W3C yet, but introducing this type seems logical)

Reply:
Yes, it`s goodStyle :)